### PR TITLE
DAPI-1480: Describe how to filter products by their quality score

### PR DESCRIPTION
**Pull request by LaurentPetard**

### CODE REVIEW

File: content/rest-api/filter.md [hunk 1]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
...
 ```
 
+### On their quality score
+::: availability versions=6.0,Serenity editions=CE,EE
+
+To filter products on their quality score, use the `quality_score` product property. You will also need to provide a `scope` and `locale` value to specify on which channel and locale you want to filter with the quality score.
+For now, this filter only accepts one operator: `IN`. It expects one or several scores, given as a list of letters. The possible values for the quality score are "A", "B", "C", "D" and "E".
+
+#### Examples
+To get the products that have a quality score "D" the `ecommerce` channel and `en_US` locale, you can use the following URL.
```
Comment (AymericPlanche) [inline content/rest-api/filter.md:215]:
```suggestion
To get the products that have a quality score "D" for the `ecommerce` channel and `en_US` locale, you can use the following URL.
```

File: content/rest-api/filter.md [hunk 2]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
 /api/rest/v1/products?search={"parent":[{"operator":"=","value":"tshirt_armor_blue"}]}
 ```
 
+### On their quality score
+::: availability versions=6.0,Serenity editions=CE,EE
```
Comment (AymericPlanche) [inline content/rest-api/filter.md:209]: do we have to add the 6.0 information now ?

**Review by AymericPlanche — APPROVED** [review]

File: content/rest-api/filter.md (hunk 2 above)
Comment (LaurentPetard) [reply content/rest-api/filter.md:209]: I'm not sure either, but I don't know how to mention that the filter will be available on CE (serenity is only EE)
In any case it's not the first. The mention "6.0" is also used for the asset families: https://github.com/akeneo/pim-api-docs/blob/master/content/php-client/resources/asset-manager/asset-families.md

File: content/rest-api/filter.md [hunk 3]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
 /api/rest/v1/products?search={"parent":[{"operator":"=","value":"tshirt_armor_blue"}]}
 ```
 
+### On their quality score
+::: availability versions=Serenity editions=CE,EE
+
+To filter products on their quality score, use the `quality_score` product property. You will also need to provide a `scope` and `locale` value to specify on which channel and locale you want to filter with the quality score.
```
Comment (fleguellec) [inline content/rest-api/filter.md:211]: you want to filter the quality score on.

File: content/rest-api/filter.md [hunk 4]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
...
+For now, this filter only accepts one operator: `IN`. It expects one or several scores, given as a list of letters. The possible values for the quality score are "A", "B", "C", "D" and "E".
```
Comment (fleguellec) [inline content/rest-api/filter.md:212]: This filter accepts one operator: `IN`. It expects one or several scores, given as a list of letters. The possible values for the quality score are "A", "B", "C", "D" and "E".

File: content/rest-api/filter.md [hunk 5]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
...
+
+#### Examples
+To get the products that have a quality score "D" for the `ecommerce` channel and `en_US` locale, you can use the following URL.
```
Comment (fleguellec) [inline content/rest-api/filter.md:215]: To get the products with a "D" for the `e-commerce` channel and `en_US` locale, you can use the following URL.

File: content/rest-api/filter.md [hunk 6]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
...
+
+```
+/api/rest/v1/products?search={"quality_score":[{"operator":"IN","value":["D"],"scope":"ecommerce","locale":"en_US"}]}
+```
+
+To get the products that have a quality score "A" or "B" for the `mobile` channel and `en_GB` locale, you can use the following URL.
```
Comment (fleguellec) [inline content/rest-api/filter.md:221]: To get the products with an "A" or "B" for the `mobile` channel and `en_GB` locale, you can use the following URL.

File: content/rest-api/filter.md [hunk 7]
```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
...
+
+```
+/api/rest/v1/products?search={"quality_score":[{"operator":"IN","value":["A","B"],"scope":"mobile","locale":"en_GB"}]}
+```
+
 ## Filter on product model properties
 
 To filter product models by one of their properties, you can use the `search` query parameter. The value given to this query parameter should be a valid JSON as shown below.
```
Comment (fleguellec) [inline content/rest-api/filter.md:229]: To filter product models with one of their properties, you can use the `search` query parameter. The value given to this query parameter should be a valid JSON as shown below.

**Review by fleguellec — CHANGES_REQUESTED** [review]

**Review by fleguellec — APPROVED** [review]

**Pull request merged**